app-arch/zstd/zstd-1.3.5.recipe

```diff
@@ -0,0 +1,139 @@
+SUMMARY="Zstandard, a fast real-time compression algorithm"
+SUMMARY_bin="Command line tools to compress/decompress .zst files"
+DESCRIPTION="Zstd, short for Zstandard, is a fast lossless compression \
+algorithm, targeting real-time compression scenarios at zlib-level and better \
+compression ratios. The zstd compression library provides in-memory \
+compression and decompression functions. It offers a very wide range of \
+compression / speed trade-off, while being backed by a very fast decoder. It \
+also offers a special mode for small data, called dictionary compression, and \
+can create dictionaries from any sample set."
+HOMEPAGE="https://facebook.github.io/zstd/"
+COPYRIGHT="2016-2018 Facebook, Inc."
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/facebook/zstd/archive/v$portVersion.tar.gz"
+SOURCE_FILENAME="zstd-$portVersion.tar.gz"
```

Comment:
```diff
-SOURCE_FILENAME="zstd-$portVersion.tar.gz"
 CHECKSUM_SHA256="d6e1559e4cdb7c4226767d4ddc990bff5f9aab77085ff0d0490c828b025e2eea"
+SOURCE_FILENAME="zstd-$portVersion.tar.gz"
 PATCHES="zstd-$portVersion.patchset"
```

(`SOURCE_FILENAME` should come after `CHECKSUM_SHA256`, not before, see [HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering).)

Reply:
Jikes :)